replica/database.cc

```diff
@@ -328,7 +328,7 @@ database::database(const db::config& cfg, database_config dbcfg, service::migrat
     }))
     , _read_concurrency_sem(max_count_concurrent_reads,
         max_memory_concurrent_reads(),
-        "_read_concurrency_sem",
+        "user",
```

Comment:
This might require some adjustment to `scylla-gdb.py`, as I think it does try to extract the "nice" name by splitting by `_` and using the second item.

Reply:
I'll check and report.